File: week4/struct0.c
#include "struct.h"
#include <stdlib.h>
#include <stdio.h>

int main(void) {
    int enrollment;
    printf("Enrollment: "); 
    scanf("%d", &enrollment);
    student students[enrollment];
    for (int i = 0; i < enrollment; i++) {
        students[i].name = "Gabriel";
        students[i].dorm = "01";
        printf("name: %s; dorm:%s\n", students[i].name, students[i].dorm);
    }

}

File: week6/struct0.py
students = []
for i in range(3):
    name = input("Name: ")
    dorm = input("Dorm: ")
    students.append({
        "name": name,
        "dorm": dorm
    })

print(students)

for student in students:
    print(f"{student['name']} is in dorm {student['dorm']}")
# below is the same
for student in students:
    name, dorm = student.items()
    print(f"{name[1]} is in dorm {dorm[1]}")